ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/StreamPublisher.java

```diff
@@ -40,7 +40,7 @@ public static Buffer toJsonMsg(final Buffer responseLine, final boolean stripArr
         end -= 1;
       }
     }
-    if (responseLine.getByte(end) == (byte) ',') {
+    if (end > 0 && responseLine.getByte(end) == (byte) ',') {
```

Comment:
Another small bug. The prior couple of modifications may strip off the square brackets. If those are the only thing in the buffer, we're left with an empty string, and we were getting an out-of-bounds exception here. Simple enough to avoid checking if an empty buffer ends in a comma.